src/components/refactor/dialogs/two-windings-transformer-creation/ratio-tap-changer-pane/ratio-tap-changer-pane-buttons.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { Button, Grid } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import {
+    HIGH_TAP_POSITION,
+    LOW_TAP_POSITION,
+    RATIO_TAP_CHANGER,
+} from 'components/refactor/utils/field-constants';
+import React from 'react';
+import { useWatch } from 'react-hook-form';
+import { FormattedMessage } from 'react-intl';
+
+const useStyles = makeStyles((theme) => ({
+    center: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+}));
+
+const RatioTapChangerPaneButtons = ({
```

Comment:
Maybe only one component to mutualize RatioTapChangerPaneButtons and PhaseTapChangerPaneButtons ?
but maybe in another us

Reply:
I'm actually doing this part in an other US